Resources/Private/JavaScript/src/state/selectedInspectorViewState.ts

```diff
@@ -0,0 +1,8 @@
+import { atom } from 'recoil';
+
+const selectedInspectorViewState = atom<null | 'asset' | 'assetCollection' | 'tag'>({
```

Comment:
Should we introduce a type/constants? Could grow further and we wouldn't need the actual string in other files.